code/__HELPERS/_lists.dm

```diff
@@ -231,14 +231,12 @@
 /*
  * Sorting
  */
-/*

```

Comment:
Because we have `reverseRange()` instead. https://github.com/tgstation/tgstation/blob/master/code/__HELPERS/_lists.dm#L394